README.md

```diff
@@ -57,8 +57,10 @@ Here's a rough roadmap:
 ### Platforms
 
 * [x] Windows
-* [ ] Mac
-* [ ] Linux
+* [x] Mac
+* [x] Linux
+
+All platforms support OpenGL. Additionally, Windows defaults to Direct3D 11 and MacOS defaults to Metal 2.
```

Comment:
macOS doesn’t support the OpenGL backend, because we require 4.3 and macOS only ships with 4.1.

Reply:
Fixed.